packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -343,11 +354,14 @@ const Step4 = () => (
     </div>
 );
 
-const Step5 = () => (
+const Step5 = ({ is_mobile }) => (
     <div className='joyride-content'>
-        <div className='joyride-content__left'>
-            <Localize i18n_default_text='Step 5 :' />
-        </div>
+        {is_mobile && (
```

Comment:
@rupato-deriv in the current case we can use component <MobileWrapper>

Reply:
done ✅